packages/node_modules/@cerebral/firebase-admin/package.json

```diff
@@ -10,7 +10,7 @@
   },
   "dependencies": {
     "faye-websocket": "^0.11.1",
-    "firebase": "^3.9.0",
+    "firebase": "^5.4.3",
```

Comment:
Probably worth to put it in peerDeps, so the user controls used versions.

Reply:
Yeah, thought about it... but is it correct usage? Cause then we do not control when the API might break due to breaking in firebase. I think this is most correct, but most maintenance of course

Reply:
We still control by defining range for what we know works for us

Reply:
Hm, I thought peerDependency was when lib did not explicitly require the host library... like middleware stuff... but made the change. Does not matter to me 😄